src/masternodes/balances.h

```diff
@@ -180,6 +211,34 @@ struct CAccountToAccountMessage {
     }
 };
 
+struct CAnyAccountsToAccountsMessage {
+    std::map<CScript, CBalances> from; // from -> balances
+    std::map<CScript, CBalances> to; // to -> balances
```

Comment:
Define an custom type for `std::map<CScript, CBalances>`, it is used in a lot of places.

Reply:
Done.